src/cache_cf.cc

```diff
@@ -1115,18 +1115,19 @@ parseTimeUnit(const char *unitName, std::chrono::nanoseconds &ns)
 }
 
 static std::chrono::nanoseconds
-CheckTimeValue(const double value, const std::chrono::nanoseconds &unit)
+ConvertToNanoSeconds(const double value, const std::chrono::nanoseconds &unit)
```

Comment:
This change is a out of scope, but I like the direction of this change.

For better consistency with such recent code as ToSBuf(), ToSBufList(), and FromNanoseconds() as well as to follow the "less ink" principle and to fix CamelCasing (nanosecond is one word rather than two), I suggest:

```suggestion
ToNanoseconds(const double value, const std::chrono::nanoseconds &unit)
```

Reply:
done